src/status_im/subs/market.cljs

```diff
@@ -0,0 +1,92 @@
+(ns status-im.subs.market
+  (:require
+    [re-frame.core :as rf]))
+
+
+(rf/reg-sub
+ :market/tokens
```

Comment:
Even though it's just the mock data, I'd suggest to normalize the market data before it's stored in the db (separately store the order of the tokens as a vector of ids and the token data as a map). This will optimize the rendering of the data in a flat-list and allow easier retrieval of the token data in case we'll have a screen for token details.

Reply:
Totally agree that when data stored in the db it should be normalized. As I mentioned in previous PR we already have a `status-go`  branch with endpoints to get this data so I'm starting the related work in another branch. It will include data normalization.
So if you don't mind I'd prefer to leave data in this PR "as is" since it is not even in rf-db. And all the improvements on rf-db structure and corresponding UI changes will happen in next PR.
wdyt?

Reply:
yeah, sounds good. Just wanted to mention this to make sure we consider it for the implementation